deploy/example_catalog/cr-app-tensorflow-notebook.json

```diff
@@ -45,15 +59,20 @@
                 }
             }
         ],
-        "defaultConfigPackage": null,
+        "defaultConfigPackage": {
+            "packageURL" : "http://16.143.21.22/thirdparty/k8sapps/notebook/appconfig-1.8.tgz"
+        },
         "roles": [
             {
+                "imageRepoTag": "docker.io/rsupekar/kd-notebook:latest",
```

Comment:
What was the base for this docker image ? We need to use the docker image that saurabh has.